packages/ragbits-core/src/ragbits/core/prompt/prompt.py

```diff
@@ -181,6 +183,44 @@ def list_few_shots(self) -> ChatFormat:
             result.append({"role": "assistant", "content": assistant_message})
         return result
 
+    def list_images(self) -> list[dict]:
+        """
+        Returns the schema of the list of images compatible with llm apis
+        Returns:
+            list of dictionaries
+        """
+        image_prompt_list = (
+            [
+                {
+                    "type": "image_url",
+                    "image_url": {"url": f"data:image/png;base64 {base64.b64encode(image).decode('utf-8')}"},
+                }
+                for image in self.images
+            ]
+            if self.images
+            else []
+        )
+        return image_prompt_list
+
+    def get_user_message(self) -> ChatFormat:
```

Comment:
Despite the name, this is not a getter for the user message, so this may be confusing to our users. Either way, I would make this method private - it seems to be an internal method only meant to be used by the `chat` method

Reply:
has been already removed - processing of image input has been moved to level of specific llm client